meta/model/src/main/kotlin/poetUtils.kt

```diff
@@ -14,10 +14,11 @@ fun FunctionModel.asLambdaTypeName(): LambdaTypeName =
         returnType
     )
 
-const val PACKAGE = "trikita.anvil"
+const val PACKAGE = "dev.inkremental"
 const val ROOT_VIEW_SCOPE = "RootViewScope"
 
-val ANY_N: ClassName = ANY.copy(nullable = true)
+val ANY_N_NULLABLE: ClassName = ANY.copy(nullable = true)
+val ANY_N: ClassName = ANY.copy(nullable = false)
```

Comment:
`ANY_N` was already nullable, as non-null `ANY` already exists in the kotlinpoet. Let's leave these fields as is and replace `ANY_N` usages with `ANY` where appropriate.

Reply:
Oh, now I get what N stands for ;)